test/test_playbooks/tasks/repository.yml

```diff
@@ -5,7 +5,6 @@
     - product_name: "Test Product"
     - repository_name: "Test Repository"
     - repository_content_type: "yum"
-    - repository_label: "just_a_test_repo"
```

Comment:
@manuelbonk you originally introduced the `label` feature in #160, but we think the label setting should happen in the `repository.yml` playbook, not the task file, as this otherwise breaks the `default(omit)` you have below. Thoughts?

Reply:
I am completely :+1: here.

Reply:
You're right, I'll move it to the `repository.yml` playbook.